Lesson_3/calculator/calc.js

```diff
@@ -0,0 +1,39 @@
+document.addEventListener('DOMContentLoaded', () => {
+   const whatMathOperation = prompt('Яку з доступних дій (add +, sub -, mult *, div /) бажаєте виконати?');
+
+   if (whatMathOperation !== '+' && whatMathOperation !== 'add'
+   && whatMathOperation !== '-' && whatMathOperation !== 'sub'
+   && whatMathOperation !== '*' && whatMathOperation !== 'mult'
+   && whatMathOperation !== '/' && whatMathOperation !== 'div') {
+      return alert('Спробуйте вибрати одну з запропонованих дій:\n add\n +\n sub\n -\n mult\n *\n div\n /');
+   }
+
+   const number1 = Number(prompt('Введіть перше число для дії'));
+
+   if (number1 !== number1) {
+      return alert('Ви ввели не число! Спробуйте ще раз!');
+   }
+
+   const number2 = Number(prompt('Введіть друге число для дії'));
+
+   if (number2 !== number2) {
+      return alert('Ви ввели не число! Спробуйте ще раз!');
+   }
+
+   const resultAddTwoNum = number1 + number2;
+   const resultSubTwoNum = number1 - number2;
+   const resultMultTwoNum = number1 * number2;
+   const resultDivTwoNum = number1 / number2;
```

Comment:
This is great idea, in general. But overhead in the current case. You need only one result before alert. All other calculations are redundant

Reply:
moved in `if`s